isstools/gui.py

```diff
@@ -501,6 +502,22 @@ def save_trajectory(self):
             else:
                 print('\n.txt is not a valid name')
 
+    def plot_traj_file(self):
+        self.traj.load_trajectory_file('/GPFS/xf08id/trajectory/' + self.comboBox.currentText())
+
+        self.figure_single_trajectory.clf()
+        self.figure_single_trajectory.add_subplot(111)
+        self.canvas_single_trajectory.draw()
```

Comment:
`draw_idle` is usually a better function to call, it lets the GUI decide when to actually do the re-rendering.